src/routes.json

```diff
@@ -218,5 +218,33 @@
         "pattern": "^/explore/studios/?$",
         "routeAlias": "/explore(?!/ajax)",
         "redirect": "/explore/studios/all"
+    },
+    {
+        "name": "microworld-art",
+        "pattern": "^/microworlds/art",
```

Comment:
A @morantsur this is my bad – I forgot we don't need to include the `^` at the beginning anymore, since we add it later in the [build process](https://github.com/LLK/scratch-www/blob/develop/bin/configure-fastly.js#L64-L72). Do you mind removing those before I merge? Sorry!